<launch>
    <node pkg="cam_rad_fusion" type="fusion" name="fusion" output="screen" />
    <node pkg="cam_rad_fusion" type="dynamic_reconfigure" name="dynamic_reconfigure" output="screen" />
    <rosparam file= "$(find cam_rad_fusion)/config/params.yaml" command="load" />
</launch>